test/spec/CodeEditor.spec.js

```diff
@@ -267,6 +266,64 @@ return
 
     });
 
+
+    it('should call onLint with errors', function(done) {
+      const initalValue = '= 15';
+      const onLint = sinon.spy();
+
+      const editor = new FeelEditor({
+        container,
+        value: initalValue,
+        onLint
+      });
+
+      const cm = editor._cmEditor;
+
+      // when
+      forceLinting(cm);
+
+      // then
+      // update done async
```

Comment:
I probably don't have enough context, but why can't we simply `await forceLinting` instead of having such a timeout hack?

Reply:
Unfortunately not, the `forceLinting` is not async :/

We could make it prettier with an update listener, so we don't need the timeout